{"logType":"1","productName":"爬虫网络请求","handle_person":"请求地址：https://www.ppoletrangers.interieur.gouv.fr/?motif=renetu","action":"使用代理请求","detail":"ERROR:HTTPSConnectionPool(host='www.ppoletrangers.interieur.gouv.fr', port=443): Max retries exceeded with url: /?motif=renetu (Caused by ProxyError('Cannot connect to proxy.', NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f285c41eba8>: Failed to establish a new connection: [Errno 111] Connection refused',)))","handle_date":"2019-02-22 14:24:18","level":"INFO"}
{"logType":"1","productName":"爬虫网络请求","handle_person":"请求地址：https://www.ppoletrangers.interieur.gouv.fr/?motif=renetu","action":"使用代理请求","detail":"ERROR:HTTPSConnectionPool(host='www.ppoletrangers.interieur.gouv.fr', port=443): Max retries exceeded with url: /?motif=renetu (Caused by ProxyError('Cannot connect to proxy.', timeout('_ssl.c:629: The handshake operation timed out',)))","handle_date":"2019-02-22 14:25:37","level":"INFO"}
{"logType":"1","productName":"爬虫网络请求","handle_person":"请求地址：https://www.ppoletrangers.interieur.gouv.fr/?motif=renetu","action":"使用代理请求","detail":"ERROR:HTTPSConnectionPool(host='www.ppoletrangers.interieur.gouv.fr', port=443): Read timed out.","handle_date":"2019-02-22 14:29:31","level":"INFO"}
{"logType":"1","productName":"爬虫网络请求","handle_person":"请求地址：https://www.ppoletrangers.interieur.gouv.fr/?motif=renetu","action":"使用代理请求","detail":"ERROR:HTTPSConnectionPool(host='www.ppoletrangers.interieur.gouv.fr', port=443): Max retries exceeded with url: /?motif=renetu (Caused by ProxyError('Cannot connect to proxy.', timeout('_ssl.c:629: The handshake operation timed out',)))","handle_date":"2019-02-22 14:37:07","level":"INFO"}
{"logType":"1","productName":"爬虫网络请求","handle_person":"请求地址：https://www.ppoletrangers.interieur.gouv.fr/?motif=renetu","action":"使用代理请求","detail":"ERROR:HTTPSConnectionPool(host='www.ppoletrangers.interieur.gouv.fr', port=443): Max retries exceeded with url: /?motif=renetu (Caused by ProxyError('Cannot connect to proxy.', timeout('timed out',)))","handle_date":"2019-02-22 14:38:32","level":"INFO"}
{"logType":"1","productName":"爬虫网络请求","handle_person":"请求地址：https://www.ppoletrangers.interieur.gouv.fr/?page=ident","action":"使用代理请求","detail":"ERROR:HTTPSConnectionPool(host='www.ppoletrangers.interieur.gouv.fr', port=443): Max retries exceeded with url: /?page=ident (Caused by ProxyError('Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response',)))","handle_date":"2019-02-22 14:39:13","level":"INFO"}
{"logType":"1","productName":"爬虫网络请求","handle_person":"请求地址：https://www.ppoletrangers.interieur.gouv.fr/?page=ident","action":"使用代理请求","detail":"ERROR:HTTPSConnectionPool(host='www.ppoletrangers.interieur.gouv.fr', port=443): Max retries exceeded with url: /?page=ident (Caused by ProxyError('Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response',)))","handle_date":"2019-02-22 14:40:17","level":"INFO"}
{"logType":"1","productName":"爬虫网络请求","handle_person":"请求地址：https://www.ppoletrangers.interieur.gouv.fr/?page=ident","action":"使用代理请求","detail":"ERROR:HTTPSConnectionPool(host='www.ppoletrangers.interieur.gouv.fr', port=443): Max retries exceeded with url: /?page=ident (Caused by ProxyError('Cannot connect to proxy.', timeout('_ssl.c:629: The handshake operation timed out',)))","handle_date":"2019-02-22 14:41:35","level":"INFO"}
{"logType":"1","productName":"爬虫网络请求","handle_person":"请求地址：https://www.ppoletrangers.interieur.gouv.fr/?page=ident","action":"使用代理请求","detail":"ERROR:HTTPSConnectionPool(host='www.ppoletrangers.interieur.gouv.fr', port=443): Max retries exceeded with url: /?page=ident (Caused by ProxyError('Cannot connect to proxy.', NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f285c34a320>: Failed to establish a new connection: [Errno 111] Connection refused',)))","handle_date":"2019-02-22 14:41:40","level":"INFO"}
{"logType":"1","productName":"爬虫网络请求","handle_person":"请求地址：https://www.ppoletrangers.interieur.gouv.fr/?motif=renetu","action":"使用代理请求","detail":"ERROR:HTTPSConnectionPool(host='www.ppoletrangers.interieur.gouv.fr', port=443): Read timed out.","handle_date":"2019-02-22 14:47:29","level":"INFO"}
{"logType":"1","productName":"爬虫网络请求","handle_person":"请求地址：https://www.ppoletrangers.interieur.gouv.fr/?page=rdv+post","action":"使用代理请求","detail":"ERROR:HTTPSConnectionPool(host='www.ppoletrangers.interieur.gouv.fr', port=443): Max retries exceeded with url: /?page=rdv+post (Caused by ProxyError('Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response',)))","handle_date":"2019-02-22 14:51:45","level":"INFO"}
{"logType":"1","productName":"爬虫网络请求","handle_person":"请求地址：https://www.ppoletrangers.interieur.gouv.fr/?motif=renetu","action":"使用代理请求","detail":"ERROR:HTTPSConnectionPool(host='www.ppoletrangers.interieur.gouv.fr', port=443): Max retries exceeded with url: /?motif=renetu (Caused by ProxyError('Cannot connect to proxy.', NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f285c34a080>: Failed to establish a new connection: [Errno 111] Connection refused',)))","handle_date":"2019-02-22 14:58:11","level":"INFO"}
{"logType":"1","productName":"爬虫网络请求","handle_person":"请求地址：https://www.ppoletrangers.interieur.gouv.fr/?page=rdv+post","action":"使用代理请求","detail":"ERROR:HTTPSConnectionPool(host='www.ppoletrangers.interieur.gouv.fr', port=443): Max retries exceeded with url: /?page=rdv+post (Caused by ProxyError('Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response',)))","handle_date":"2019-02-22 15:02:16","level":"INFO"}
{"logType":"1","productName":"爬虫网络请求","handle_person":"请求地址：https://www.ppoletrangers.interieur.gouv.fr/?page=ident","action":"使用代理请求","detail":"ERROR:HTTPSConnectionPool(host='www.ppoletrangers.interieur.gouv.fr', port=443): Read timed out.","handle_date":"2019-02-22 15:09:33","level":"INFO"}
{"logType":"1","productName":"爬虫网络请求","handle_person":"请求地址：https://www.ppoletrangers.interieur.gouv.fr/?page=ident","action":"使用代理请求","detail":"ERROR:HTTPSConnectionPool(host='www.ppoletrangers.interieur.gouv.fr', port=443): Read timed out.","handle_date":"2019-02-22 15:18:33","level":"INFO"}
{"logType":"1","productName":"爬虫系统","handle_person":"ZHAO YITONG预约成功","action":"预约成功","detail":"预约时间：2019-03-01 11:00:00 预约地点：Préfecture de PoliceCentre de réception des étudiants étrangers et des chercheurs internationauxCité internationale universitaire de Paris17 Boulevard Jourdan75014 Paris - RER B ou Tramway 3a Cité Universitaire","handle_date":"2019-02-22 15:19:17","level":"INFO"}
